src/Сomponents/Button/Button.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-//import styles from './button.module.css';
 
-//
-// <Button click={} text={}/>
-//
-export function Button(props) {
-  return (
-  <button onClick={props.handleClick}>
-    {props.text}
+import styles from './button.module.css';
+
+export const Button = ({onClick, disabled, text, value}) => (
```

Comment:
its a good practise to use default values for some props.
e.g `disabled = false`

Reply:
Done